lib/global_amd.js

```diff
@@ -32,12 +32,12 @@ module.exports = function(load){
 		"  });\n" +
 		"  var define = loader.global.define;\n" +
         "  var require = loader.global.require;\n" +
-		"  var source = \"" + source + "\";\n" +
+		"  var source = \"" + (metadata.exports ? (source + 'this[\'' + metadata.exports + '\'] = ' + metadata.exports) : source)  +  "\";\n" +
 		(metadata.init ? "  var init = " + metadata.init.toString() + ";\n" : "") +
         "  loader.global.define = undefined;\n" +
         "  loader.global.module = undefined;\n" +
         "  loader.global.exports = undefined;\n" +
-        "  loader.__exec({'source': source, 'address': module.uri});\n" +
+        "  loader.__exec({'source': source, 'address': module.uri, 'metadata': {'eval': '" + (metadata.eval || '') +  "'} });\n" +
```

Comment:
Tests are failing because this changes the output. Would be better to only add this bit if `metadata.eval` is truthy.